packages/flutter/lib/src/material/material.dart

```diff
@@ -86,9 +104,18 @@ abstract class MaterialInkController {
 /// [InkSplash] and [InkHighlight] effects. To trigger a reaction on the
 /// material, use a [MaterialInkController] obtained via [Material.of].
 ///
-/// If a material has a non-zero [elevation], then the material will clip its
-/// contents because content that is conceptually printing on a separate piece
-/// of material cannot be printed beyond the bounds of the material.
+/// ## Shape
+/// The shape for material is determined by [type] and [borderRadius].
+///  - If [borderRadius] is non null, the shape is a rounded rectangle, with
```

Comment:
blank line after header and before list item

Reply:
done